persistent-sqlite/Database/Persist/Sqlite.hs

```diff
@@ -57,7 +68,17 @@ import Control.Monad.Trans.Writer (runWriterT)
 -- Instead, use 'withSqliteConn'.
 createSqlitePool :: (MonadIO m, MonadLogger m, MonadBaseControl IO m, IsSqlBackend backend)
                  => Text -> Int -> m (Pool backend)
-createSqlitePool s = createSqlPool $ open' s
+createSqlitePool = createSqlitePoolFromInfo . conStringToInfo
+{-# DEPRECATED createSqlitePool "Use createSqlitePoolFromInfo" #-}
```

Comment:
I'm not sure what the motivation is for deprecating these, I don't think we discussed that either. These string-based functions are likely sufficient for most use cases.